db/migrate/20190610093304_add_conviction_length_to_disclosure_check.rb

```diff
@@ -0,0 +1,6 @@
+class AddConvictionLengthToDisclosureCheck < ActiveRecord::Migration[5.2]
+  def change
+    add_column :disclosure_checks, :conviction_length_type, :string
+    add_column :disclosure_checks, :conviction_length, :integer, default: 0
```

Comment:
Please do not use a default. Otherwise on loading the form it will show in the input. Being an annoyance for the user to have to delete and change, or worse, making it confusing for the user.